### PR TITLE
Add run of show for sponsors.

### PR DESCRIPTION
**Pull request by ericholscher**
This is the last piece required before sending out the "1 month out",
which will include the job fair upgrade.

### CODE REVIEW

File: docs/conf/prague/2018/sponsors/prospectus.rst [hunk 1]
```diff
@@ -230,21 +230,61 @@ of our expenses must be paid up front.
...
 
-Frequently Asked Questions
-==========================
+Run of Show
+===========
+
+A quick overview of the spaces:
+
+* The *Main Ballroom* is where talks and most sponsorship activity happens. 
+* The *Unconference Room* is across from the main ballroom. This is where the Unconference & Job Fair will happen.
```
Comment (plaindocs) [inline docs/conf/prague/2018/sponsors/prospectus.rst]: s/will happen/happens/

 to match Main Ballroom?

File: docs/conf/prague/2018/sponsors/prospectus.rst [hunk 2]
```diff
@@ -230,21 +230,61 @@ of our expenses must be paid up front.
...
+
+What do I need for the job fair?
+--------------------------------
+
+The job fair will be a low key event. Every participant will have a six-foot table and two chairs, in a seperate room from the primary conference. Giveaways/flyers are welcome, but please keep your setup requirements simple.
+
+What happens with my swag items?
+--------------------------------
+
+We will have a few "swag tables" that are placed around the back of the main ballroom. This will be where sponsor and community stickers & swag will be located, so that attendees are free to pick it up. If you have a booth, you are also welcome to place swag on the booth.
```
Comment (plaindocs) [inline docs/conf/prague/2018/sponsors/prospectus.rst]: a/booth/table/ ?

**Review by plaindocs — COMMENTED** [review]
Good stuff.

**Comment by ericholscher** [comment]
👍

**Pull request merged**